VERSION

```diff
@@ -1 +1 @@
-2.0.0-alpha+006
+2.1-develop
```

Comment:
Is this the commit that will be tagged as "Release 2.0"? If so shall this version not say "2.0.0"? (I.e. change to "2.1-develop" first in the following commit?